components/src/dynamo/sglang/backend_args.py

```
@@ -186,6 +186,13 @@ class DynamoSGLangConfig(ConfigBase):
     frontend_decoding: bool = False
     sglang_trace_level: int
 
+    # Additional names this model responds to (aliases). Populated by
+    # parse_args when --served-model-name is given as a whitespace- or
+    # comma-separated string of multiple names; the first becomes the
+    # primary and the rest land here. Plumbed to register_model() so the
+    # Rust ModelManager registers the same WorkerSet under each alias.
+    served_model_aliases: List[str] = []
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)